developer_manual/client_apis/WebDAV/basic.rst

```diff
@@ -343,12 +355,49 @@ You can request properties of a folder without also getting the folder contents
 Downloading files
 -----------------
 
+.. note:: The :code:`GET` method is not defined by the WebDAV standard, this is a Nextcloud provided WebDAV extension.
+.. note:: For shared files this only works if the download permission was not denied by the sharer.
+
 A file can be downloaded by sending a :code:`GET` request to the WebDAV url of the file.
 
 .. code::
 
 	GET remote.php/dav/files/user/path/to/file
 
+.. _webdav-download-folders:
+
+Downloading folders
+-------------------
+
+.. note:: The :code:`GET` method is not defined by the WebDAV standard, this is a Nextcloud provided WebDAV extension.
+.. note:: For shared folders this only works if the download permission was not denied by the sharer.
+
+A folder can be downloaded as an archive by sending a :code:`GET` request to the WebDAV URL of the folder.
+The :code:`Accept` header must be set and contain the MIME type for ZIP archives (:code:`application/zip`) or tarballs (:code:`application/x-tar`).
+
+.. code::
+
+	GET remote.php/dav/files/user/path/to/folder
+	Accept: application/zip
+
+Optionally it is possible to only include some files from the folder in the archive by providing the files using the custom :code:`X-NC-Files` header:
+
+.. code::
+
+	GET remote.php/dav/files/user/path/to/folder
+	Accept: application/zip
+	X-NC-Files: document.txt
+	X-NC-Files: image.png
+
+As setting headers is not possible with HTML links, and this endpoint is also used for downloading folders from the Nextcloud web interface,
+it is also possible to provide this both options as query parameters.
+In this case the :code:`Accept` header value must be passed as the :code:`accept` query parameter.
+The optional files list can be provided as a JSON encoded array through the :code:`files` query parameter.
+
+.. code::
+
+	GET remote.php/dav/files/user/path/to/folder?accept=zip&files=%5B%22image.png%22%2C%22doucment.txt%22%5D
```

Comment:
For readability purposes it would be better to write the non-encoded variant

Reply:
Makes sense!

```suggestion
	GET remote.php/dav/files/user/path/to/folder?accept=zip&files=["image.png","document.txt"]
```